### ISSUE TITLE
G3 - Local search has no label

### ISSUE THREAD

**Issue by jpinsent** [ccswbs/hjckrrh#229]
Local search needs to have a label associated with its input form.

This issue is related to several other issues (#212, #170, #204)

**Comment by mmafe** [comment]
sites/all/themes/bootstrap/theme/bootstrap/bootstrap-search-form-wrapper.func.php

**Comment by jpinsent** [comment]
Local search now has a label reading "Search this site". This issue has been resolved.
